benchmark/utils/DriverUtils.swift

```diff
@@ -227,27 +231,53 @@ func stopTrackingObjects(_: UnsafeMutableRawPointer) -> Int
 #endif
 
 class SampleRunner {
```

Comment:
Fixed in https://github.com/apple/swift/pull/7810/commits/d945522a8645d8169f387aeb359248c44b1aa2dd. Thanks.